### PR TITLE
674 aab tasks service

### PR DESCRIPTION
**Pull request by HalaszPaul**

### CODE REVIEW

**Review by ovidiufilip10 — CHANGES_REQUESTED** [review]
Please remove unnecessary whitespaces from the code.

File: src/constants/api.js [hunk 1]
```diff
@@ -4,4 +4,7 @@ export const ENDPOINTS = {
 	AUTH: {
 		LOGIN: `${API}/v1/auth/login`,
 	},
+	TASKS:{
+		GET_TASKS:(workspaceId) => `${API}/v1/workspaces/${workspaceId}/tasks`,
```
Comment (cezarmocanu) [inline src/constants/api.js:8]: ALL_TASKS

File: src/services/tasks-service.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        headers: {
+            "Authorization": `Bearer ${token}`
+    },
+});
+
+if (response.status !== 200) {
+    return[];
+}
+const data = await response.json();
+return data.items;
```
Comment (cezarmocanu) [inline src/services/tasks-service.js:19]: review formatting

File: src/constants/api.js [hunk 3]
```diff
@@ -4,4 +4,7 @@ export const ENDPOINTS = {
 	AUTH: {
 		LOGIN: `${API}/v1/auth/login`,
 	},
+	TASKS:{
```
Comment (cezarmocanu) [inline src/constants/api.js:7]: Add test with use effect in test page

File: src/services/tasks-service.js (hunk 2 above)
Comment (cezarmocanu) [reply src/services/tasks-service.js:19]: data is enough because it has no items property

File: src/store/slices/tasks-slice.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import { createSlice } from "@reduxjs/toolkit";
+import taskService from "../../services/tasks-service";
+
+const taskSlice = createSlice({
+    name: "tasks",
+    initialState: {
+        items: [],
```
Comment (cezarmocanu) [inline src/store/slices/tasks-slice.js:7]: modifyt everywhere to tasks

**Review by cezarmocanu — CHANGES_REQUESTED** [review]

**Review by cezarmocanu — APPROVED** [review]

**Review by AdrianaGroza — APPROVED** [review]

**Review by OanaIv — APPROVED** [review]

**Review by MariusStan2001 — APPROVED** [review]

**Pull request merged**